src/main/java/core/basesyntax/strategy/impl/NewYearDiscountService.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.DiscountService;
+
+public class NewYearDiscountService implements DiscountService {
+
+    @Override
```

Comment:
```suggestion
public class NewYearDiscountService implements DiscountService {
    @Override
```
старайся дотримуватись одного стилю коду